lib/rules/lint-no-meta-redirect-with-time-limit.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const AstNodeInfo = require('../helpers/ast-node-info');
+const Rule = require('./base');
+
+module.exports = class noMetaRedirectWithTimeLimit extends Rule {
+  logNode({ node, message }) {
+    return this.log({
+      message,
+      line: node.loc && node.loc.start.line,
+      column: node.loc && node.loc.start.column,
+      source: this.sourceForNode(node),
+    });
+  }
+  visitor() {
+    return {
+      ElementNode(node) {
+        if (AstNodeInfo.hasAttribute(node, 'hidden')) {
+          return;
+        }
+
+        if (AstNodeInfo.hasAttributeValue(node, 'aria-hidden', 'true')) {
+          return;
+        }
+
+        const isMeta = AstNodeInfo.isElementNode(node, 'meta');
```

Comment:
`isElementNode` accept only one argument

```suggestion
        const isMeta = node.tag === 'meta';
```

Is it possible to move this assert on the top because of perf reasons? (we don't need to check element visibility if this is not `meta` node.